src/theme/media.js

```diff
@@ -2,19 +2,20 @@ import { css } from 'styled-components'
 import theme from './theme'
 
 export const media = {
-  tablet() {
-    return `@media (min-width: ${theme.breakpoints[1]}) {
-      ${css(...arguments)}
-    }`
-  },
-  desktop() {
-    return `@media (min-width: ${theme.breakpoints[2]}) {
-      ${css(...arguments)}
-    }`
-  },
-  desktopHD() {
-    return `@media (min-width: ${theme.breakpoints[3]}) {
-      ${css(...arguments)}
-    }`
-  },
+  tablet: (...args) => css`
```

Comment:
This fixes an issue where inline functions and other css blocks were not being handled correctly in the template strings.